### ISSUE TITLE
The Calendar layout Does not work anymore after update to v10

### ISSUE THREAD

**Issue by tawsbob** [directus/directus#18492]
### Describe the Bug

After select the start date on layout config I got this error on console and the entries does not appear in the calendar

```bash
ntime-core.esm-bundler-b81764e2.js:1 TypeError: Cannot read properties of undefined (reading 'value')
    at Lve (index.99d5ef4c.entry.js:52:29525)
    at Pe (index.99d5ef4c.entry.js:1894:278073)
    at index.99d5ef4c.entry.js:1894:275312
    at Proxy.map (<anonymous>)
    at hn.fn (index.99d5ef4c.entry.js:1894:275304)
    at hn.run (runtime-core.esm-bundler-b81764e2.js:1:5062)
    at get value [as value] (runtime-core.esm-bundler-b81764e2.js:1:13896)
    at hn.fn (index.99d5ef4c.entry.js:1894:275860)
    at hn.run (runtime-core.esm-bundler-b81764e2.js:1:5062)
    at get value [as value] (runtime-core.esm-bundler-b81764e2.js:1:13896)
```

### To Reproduce

Create or use one collection with date field and try to set the template to calendar with start date

### Directus Version

v10.0.0

### Hosting Strategy

Self-Hosted (Docker Image)

**Comment by paescuj** [comment]
Thanks for opening this issue! However this is a duplicate of an existing issue #18206, so I'll proceed to close this but if you think this is actually a different issue, then please do not hesitate to comment. The existing issue #18206 has been fixed by #18267 which will be available with the next release.

**Comment by tawsbob** [comment]
Thanks!
